R/plot_forecast.R

```diff
@@ -122,5 +139,7 @@ plot_forecast <- function(forecast_data,
                       location,": observed and forecasted") ,
          caption=paste0("source: ", truth_source," (observed data), ",
                         model," (forecasts)")) 
+  
+  return (graph)
```

Comment:
if we add the `plot=TRUE` argument above, let's also make this `return(invisible(graph))`.  Now that I'm saying this I'm not sure what advantage this set up has, but I have the feeling there's some edge case where the right thing to do is to explicitly print the plot in the function and then return the plot object invisibly....